x/bank/handler.go

```diff
@@ -31,13 +32,20 @@ func handleMsgLockedSend(ctx sdk.Context, k Keeper, ak types.AccountKeeper, msg
 
 	acc = ak.GetAccount(ctx, msg.To)
 	if acc == nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrUnknownAddress, "receiver account %s does not exist", msg.To)
+		acc = ak.NewAccountWithAddress(ctx, msg.To)
 	}
 
 	// ensure correct account type
 	toAcc, ok := acc.(*vesting.ManualVestingAccount)
 	if !ok {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "receiver account is not a ManualVestingAccount")
+		baseAcc := auth.NewBaseAccount(msg.To, acc.GetCoins(), acc.GetPubKey(), acc.GetAccountNumber(), acc.GetSequence())
+		unlocker, err := sdk.AccAddressFromBech32(msg.Unlocker)
+		if err != nil {
+			return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "need to specify the unlocker when initializing a new vesting account")
+		}
+		toAcc = vesting.NewManualVestingAccount(baseAcc, acc.GetCoins(), unlocker)
```

Comment:
Oh If `locked-send` is sent to a regular existing address, it would end up here which is not what we want.

Reply:
I think this ManualVestingAccount creation part should go inside the previous if statement.